pom.xml

```diff
@@ -130,6 +131,11 @@
         <artifactId>byteman-bmunit5</artifactId>
         <version>${byteman.version}</version>
       </dependency>
+       <dependency>
+          <groupId>org.jboss</groupId>
+          <artifactId>jboss-transaction-spi</artifactId>
+          <version>${jboss-transaction-spi.version}</version>
+        </dependency>
```

Comment:
@luigidemasi why do you need to add `jboss-transaction-spi` here?

Reply:
because jboss-transaction-spi has been removed from spring boot bom